<!-- Launches transforms publisher. -->
<launch>
  <arg name="transforms_path" doc="Sensor transforms file path."/>

  <arg name="parent_frame_id" default="cepton" doc="Parent transform frame."/>
  <arg name="rate" default="10" doc="Publisher rate."/>

  <node pkg="cepton_ros" type="transforms_node.py" name="cepton_transforms" output="screen">
    <param name="parent_frame_id" value="$(arg parent_frame_id)"/>
    <param name="rate" value="$(arg rate)"/>
    <param name="transforms_path" value="$(arg transforms_path)"/>
  </node>
</launch>
